### PR TITLE
Update docfx.json

### PR DESCRIPTION
**Pull request by dovholuknf**
updating the repo link for github. pretty sure this will link you to the page you're viewing.

### CODE REVIEW

File: docfx_project/docfx.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
         "branch": "master",
         "apiSpecFolder": "apidoc"
       }
-      , "_github_link":"https://github.com/openziti"
+      , "_github_link":"https://github.com/openziti/ziti-doc"
```
Comment (smilindave26) [inline docfx_project/docfx.json:90]: Is this the link in the top right?  I kinda like it better going to org root than down into the ziti-doc repo.  The 'Improve this doc' looks right...

Comment (dovholuknf) [reply docfx_project/docfx.json:90]: It's an option, sure... Before you would get linked directly to the page you were viewing. I'll leave this open for a bit to see if there's any other opinions in either direction.

Comment (smilindave26) [reply docfx_project/docfx.json:90]: 'Improve this doc' already links directly to the page you are viewing.

**Comment by dovholuknf** [comment]
thanks for talking me off that ledge :D where's the facepalm emoji?

**Pull request closed without merging**